### PR TITLE
create popup

### PR DESCRIPTION
**Pull request by brightbrain20**
- created popup dynamically
- stored card details in data.js file

### CODE REVIEW

**Review by AmaduKamara — COMMENTED** [review]
# Status: Invalid Submission ❌ ❌ ❌

Hi @brightbrain20,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is having a conflict

- [x] Kindly resolve the conflict this branch has so linter checks are visible and then your project can be reviewed 👍🏽

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: index.html [hunk 1]
```diff
@@ -121,50 +123,50 @@ <h1>Hey There. I'm Bright</h1>
             Ruby
           </li>
         </ul>
-        <a href="#1" class="Multi-Post-project button">See Project</a>
+        <a href="#1" id="one"  class="Multi-Post-project cards button">See Project</a>
       </div>
     </div>
   </section>
   <section class="work-container">
```
Comment (zieeco) [inline index.html:91]: - Hi Team, The [contents](https://github.com/brightbrain20/my-portfolio/blob/182e1b741a71fd9faf656598a6ae143bca4c8def/index.html#L131-L252) in the project section should be added dynamically using JS. Your project section is still hardcoded. please, let's fix this.

Tips: One lazy way to do it is to create a function that will display all the contents in the works section. eg

```
function populateProjets() {
 the targetted works innerHTML = `all of it content........`
}
populateProjects();

```
- The section should look something similar to this in the HTML file 

```
<section class="work-container">
        <!-- displayed dynamically with JS  -->
</section>

```

- Your popUp is great, just remaining the project section in your HTML file :+1:

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Nice PopUP :100: 

## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ const removeBtn = document.querySelector('.remove-btn');
 const mobileNav = document.querySelector('#mobile-navigation');
 const mobileNavLinks = document.querySelectorAll('.mobile-item');
 const popupContainer = document.querySelector('.popup');
+const work = document.querySelector('.work-container');
+const about = document.querySelector('#about');
+
 
 const navToggle = () => {
   mobileNav.classList.toggle('active');
```
Comment (topeogunleye) [inline index.js:12]: - [ ] [OPTIONAL] Your modal doesn't show everything that's on it. please fix it. Perhaps use a vertical overflow to allow a scroll.
- [ ] [OPTIONAL] Kindly make sure your buttons have spacing between them like on the Figma Template

|  | |
| ------------- | ------------- |
| Your Porfolio | image  |
| Template Design | image  |

**Review by topeogunleye — APPROVED** [review]
### Hi @brightbrain20, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**